src/components/social-media-icon/index.js

```diff
@@ -6,6 +6,18 @@ import classNames from '@components/social-media-icon/social-media-icon.module.s
 const SocialMediaIcon = (props) => {
   const { id, type } = props;
 
+  let socialUserId = id;
+
+  if (
+    socialUserId.includes('linkedin') &&
```

Comment:
if socialUserId is undefined/null for some reason, then all string fn i.e .includes can give error. 
Instead can we handle undefined/null case at  outser level only i.e line 10 with single if(socialUserId){} 
Thank you.

Reply:
Shubham same has resolved now.
Thanks